### PR TITLE
Comprehensive Call Cache Capoeirae!

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+        File balanca = "balanca.txt"
+    }
+}
+
+task modify_file_gcs {
+    Boolean ready
+    String file_path_raw
+    command {
+        echo "override" > tmp
+        gsutil cp tmp ${file_path_raw} && sleep 2
```
Comment (kshakir) [inline src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl]: Remove this `sleep 2`.

**Comment by kshakir** [comment]
👍 once the extra sleep is removed

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/159/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    # Call cache the made files, creating some copies:
+    call make_files as make_files_cached { input: ready = make_files.done }
+
+    # Make sure read_files is in the call cache:
+    call read_files { input: ready = true, a = make_files.bananeira, b = make_files.balanca }
+
+    # Modify that file! Oh so crafty!
+    call modify_file_gcs { input: ready = read_files.done, file_path_raw = make_files.bananeira }
+
+    # The file inputs are different., shouldn't hit the call cache.
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl:13]: ., => ,

File: src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl [hunk 3]
```diff
@@ -0,0 +1,345 @@
+workflow call_cache_capoeira {
+    # Make some files:
+    call make_files { input: ready = true }
+    # Call cache the made files, creating some copies:
+    call make_files as make_files_cached { input: ready = make_files.done }
+
+    # Make sure read_files is in the call cache:
+    call read_files { input: ready = true, a = make_files.bananeira, b = make_files.balanca }
+
+    # Modify that file! Oh so crafty!
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_jes.wdl:10]: good to know we'll have a test that depends on not hashing outputs... 🙂

**Comment by mcovarr** [comment]
:+1: Very thorough!  Ironic the only backend not getting the Capoeira treatment is Amazon... 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/159/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Review by mcovarr — APPROVED** [review]

**Pull request merged**